src/html/pages/abc.html

```diff
@@ -0,0 +1,74 @@
+<!--
```

Comment:
Should we remove abc.html from the repo?

Or do you want to put in a tests dir?